Apps/APN-UIKit/APN UIKit/AppDelegate.swift

```diff
@@ -11,11 +11,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         // Override point for customization after application launch.
 
-        // Register for remote notifications using APN. On successful registration,
-        // didRegisterForRemoteNotifications delegate method will be called and it
-        // provides a device token. In case, registration fails then
-        // didFailToRegisterForRemoteNotifications will be called.
         initializeCioAndInAppListeners()
+
+        // Register a 2nd push click listener besides the Customer.io SDK.
+        // Tests that the SDK is able to handle push clicks when there are multiple click listeners.
```

Comment:
Are we going to push these changes to `main`? Could this be confusing for customers looking into sample when not using multiple click listeners?

Reply:
Yes, I was planning on pushing these changes to `main` so we can continue to test this important edge case of having multiple push services installed. 

I can see how this could make our example apps more complex to read. I think writing some comments directed towards customers would help. If you have a different idea, please share.

Reply:
I'm good with comments to keep it easier for customers to understand 👍🏻